workspace-manager-clienttests/tools/render-config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+VAULT_TOKEN=${1:-$(cat "$HOME"/.vault-token)}
+WM_APP_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/dev/workspace/app-sa
+WM_APP_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/../rendered/service-account.json
+USER_DELEGATED_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/firecloud/dev/common/firecloud-account.json
+USER_DELEGATED_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/../rendered/user-delegated-service-account.json
+JANITOR_CLIENT_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/integration/tools/crl_janitor/client-sa
+JANITOR_CLIENT_SERVICE_ACCOUNT_OUTPUT_PATH="$(dirname $0)"/../rendered/janitor-client-sa-account.json
```

Comment:
Do you need the Janitor client SA?

Reply:
This is part of the render-config.sh that was used by WSM. I just copied it and make it work inside the test subdirectory. I'd just leave it there.